src/ddsrt/include/dds/ddsrt/event.h

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright(c) 2006 to 2020 ADLINK Technology Limited and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Eclipse Distribution License
+ * v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+#ifndef EVENT_H
+#define EVENT_H
+
+#if defined (__cplusplus)
+extern "C" {
+#endif
+
+/*currently supported monitorable quantities*/
+enum MONITORABLE_TYPE {
```

Comment:
Please prepend with `DDSRT_`. Also, we only use uppercase for macro's, types are lowercase (prefix becomes `ddsrt_`) and often we create a `typedef` for it.